frontend/probloom/src/containers/ProblemSet/ProblemSetSearch/ProblemSetSearch.tsx

```diff
@@ -40,20 +40,122 @@ const creatorOptions = [
+    { key: 'tag-history', text: 'history', value: 'tag-history' },
+    { key: 'tag-linguistics', text: 'linguistics', value: 'tag-linguistics' },
+    { key: 'tag-aesthetics', text: 'aesthetics', value: 'tag-aesthetics' },
+    {
+      key: 'tag-religious-studies',
+      text: 'religious-studies',
+      value: 'tag-religious-studies',
+    },
+  ],
+  'tag-social-science': [
+    { key: 'tag-psychology', text: 'psychology', value: 'tag-psychology' },
+    { key: 'tag-economics', text: 'economics', value: 'tag-economics' },
+    {
+      key: 'tag-political-science',
+      text: 'political-science',
+      value: 'tag-political-science',
+    },
+    { key: 'tag-sociology', text: 'sociology', value: 'tag-sociology' },
+    {
+      key: 'tag-anthropology',
+      text: 'anthropology',
+      value: 'tag-anthropology',
+    },
+    { key: 'tag-geography', text: 'geography', value: 'tag-geography' },
+    {
+      key: 'tag-social-welfare',
+      text: 'social-welfare',
+      value: 'tag-social-welfare',
+    },
+  ],
+  'tag-science': [
+    { key: 'tag-statistics', text: 'statistics', value: 'tag-statistics' },
+    { key: 'tag-mathematics', text: 'mathematics', value: 'tag-mathematics' },
+    { key: 'tag-biology', text: 'biology', value: 'tag-biology' },
+    { key: 'tag-physics', text: 'physics', value: 'tag-physics' },
+    { key: 'tag-chemistry', text: 'chemistry', value: 'tag-chemistry' },
+    { key: 'tag-astronomy', text: 'astronomy', value: 'tag-astronomy' },
+  ],
+  'tag-engineering': [
+    {
+      key: 'tag-mechanical-engineering',
+      text: 'mechanical-engineering',
+      value: 'tag-mechanical-engineering',
+    },
+    {
+      key: 'tag-electrical-and-electronic-engineering',
+      text: 'electrical-and-electronic-engineering',
+      value: 'tag-electrical-and-electronic-engineering',
+    },
+    {
+      key: 'tag-computer-engineering',
+      text: 'computer-engineering',
+      value: 'tag-computer-engineering',
+    },
+    {
+      key: 'tag-materials-engineering',
+      text: 'materials-engineering',
+      value: 'tag-materials-engineering',
+    },
+    {
+      key: 'tag-nuclear-engineering',
+      text: 'nuclear-engineering',
+      value: 'tag-nuclear-engineering',
+    },
+    {
+      key: 'tag-industrial-engineering',
+      text: 'industrial-engineering',
+      value: 'tag-industrial-engineering',
+    },
+    {
+      key: 'tag-chemical-engineering',
+      text: 'chemical-engineering',
+      value: 'tag-chemical-engineering',
+    },
+    {
+      key: 'tag-biological-engineering',
+      text: 'biological-engineering',
+      value: 'tag-biological-engineering',
+    },
+  ],
+  'tag-etc': [
+    {
+      key: 'tag-art',
+      text: 'art',
+      value: 'tag-art',
+    },
+    {
+      key: 'tag-music',
+      text: 'music',
+      value: 'tag-music',
+    },
+    {
+      key: 'tag-business',
+      text: 'business',
+      value: 'tag-business',
+    },
+  ],
+};
+
```

Comment:
I think tags and sub-tags also have to be received from backend. I will change this part after views implemented.